test/box/fix-not-deleting-unix-socket-path.lua

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env tarantool
```

Comment:
I just got to running the test and:
```
Start failed: builtin/box/console.lua:865: failed to create server unix/:/Users/gerold/Work/Repositories/tarantool/test/var/001_box/fix-not-deleting-unix-socket-path.socket-admin: No buffer space available
```
I would suggest to shorten the name. `box.cfg-unix-socket-del.test.lua` produces `unix/:/Users/gerold/Work/Repositories/tarantool/test/var/001_box/box.cfg-unix-socket-del.socket-admin` which is 101 bytes and should fit.

Reply:
Fixed, but i rename to `box-cfg-unix-socket-del.test.lua`, because in case we use `box.cfg-unix-socket-del.test.lua`
it's fails with error `builtin/box/console.lua:865: failed to create server unix/:/home/evgeny/TARANTOOL/IPROTO/tarantool/test/var/001_box/box.socket-admin: Address already in use`

Reply:
That looks like a bug for test-run. It truncated the test name at `.` which does not seem right. Could you please create a ticket in test-run repository?

Reply:
Ok, https://github.com/tarantool/test-run/issues/300